charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -112,18 +112,6 @@ data:
         airflow:
           useAstroSecurityManager: true
           webserver:
-            {{ if .Values.global.kubedEnabled }}
```

Comment:
Are we sure that we want to delete this part here ?

Maybe we want it in case the `config-syncer` is enabled ?

I didn't change this part in my PR because I didn't know about it. My understanding here is that if kubed isn't enabled, then Houston uses its own jwt Signing certificate instead of the one generated by the Helm release (contained in the secret mounted here).

In my demo and tests, it worked fine since my Airflow deployments have access to Houston and can retrieve it by sending requests to the resolver, but I believe it wouldn't work if Airflow deployment can't talk to Houston (Private CA situations).

I think the safest solution here would be to keep this code but change:
```yaml
{{ if .Values.global.kubedEnalbed }}
# becomes
{{ if .Values.astronomer.configSyncer.enabled }}
```

Reply:
Good catch. I don't know any of the behaviors around the questions you raised but we should figure that out before choosing a path forward here. I think your suggestion would work. @rob-1126 any ideas here regarding private CA behaviors with airflow?

Reply:
Yes, we actually had to issue updated versions to all currently maintained versions of Airflow about a month ago to avoid using jwk handling logic that fetched the secret via http because of private CAs. I'd like to see certs in the extraCaCerts bundle be replicated by houston into the airflow namespaces and the work done in airflow to mount them into webserver (and wouldn't be apposed to seeing them mounted in other containers).

Reply:
I will say that if your replicating extraCaCerts for some installs, I'm very inclined to say "we might as well replicate the jwk in the same way for 100% of installs and not fetch via http at all"